packages/messaging/lib/MessageType.ts

```diff
@@ -48,6 +48,8 @@ export enum MessageType {
 
   DlcCancelV0 = 52172,
 
+  DlcCloseV0 = 11111, // TODO
```

Comment:
There is some ongoing but un-decided discussion here
https://github.com/discreetlogcontracts/dlcspecs/pull/170/files#r674408145